docs/release-notes.md

```diff
@@ -3,6 +3,43 @@ nav_order: 9
 ---
 
 # Release notes
+## Butane 0.15.0 (2022-06-21)
+
+### Features
+
+- Add arm64 support for butane container
+- Add Flatcar support
+- Stabilize OpenShift spec 4.10
+- Add GRUB password support
+- Add TranslationSet.Map() to the Go API
+- Add AddFromCommonObject() to the Go API
+
+### Bug fixes
```

Comment:
We should release-note 5996fbe48c89da11d97dc0aee56ce00cd7d47c33.  (And separately, we should mention its Go API break.)

Reply:
Would we want two bullet points for this? 

Or just one in 'Breaking Changes' section as we have done in the past?